include/jemalloc/internal/ph.h

```diff
@@ -414,6 +414,73 @@ ph_remove(ph_t *ph, void *phn, size_t offset, ph_cmp_t cmp) {
 	}
 }
 
+JEMALLOC_ALWAYS_INLINE void
+ph_queue_push(void *phn, void **bfs_queue, uint16_t *rear, size_t *queue_size,
```

Comment:
nit: Since this is only used for the enumerate purpose, maybe name it ph_enumerate_queue_...